eitprocessing/binreader/sequence.py

```diff
@@ -533,6 +542,19 @@ def _read_frame(
 
         return event_marker
 
+    def _parse_medibus_data(
+        self, index: int, data: list, waveform_data: dict[str, NDArray]
+    ):
+        for key, value in zip(medibus_field_names, data):
+            # some fields have a value of -1000 when not available; others a very large negative
+            # number which varies, but seems to always be lower than -3e37
```

Comment:
Also, I noticed that some fields have all 0 values (`leak minute volume [L/min]` and `'spontaneous respiratory rate [/min]` in the test data set I used). I am assuming that those also haven't been measured. Maybe there should also be a line that if all values are 0, then replace by `nan` instead?

Has to be different from what was done below, because those are done on a per value level, and this needs to be evaluated on an entire field.

Reply:
Those are expected 0 values. Leak minute volume is supposed to be 0, as it represents leakage from the system or the patient. Positive values are worrisome. 0 values are good, and are not the same as 'no data'.

As for the spontaneous respiratory rate, this is also expected to be 0 is some patients, and positive in others. Knowing it is 0 is actual information, and not the same as 'no data'.